packages/@react-spectrum/s2/stories/Checkbox.stories.tsx

```diff
@@ -17,13 +17,15 @@ import {style} from '../style/spectrum-theme' with {type: 'macro'};
 const meta: Meta<typeof Checkbox> = {
   component: Checkbox,
   parameters: {
-    layout: 'centered'
+    layout: 'centered',
+    chromatic: {delay: 10000}
```

Comment:
Note: will remove this delay soon, just added as part of Chromatic support debugging for S2 font vertical centering